packages/react/src/components/GroupedList/GroupedListV2.base.tsx

```diff
@@ -396,14 +406,27 @@ export const GroupedListV2FC: React.FC<IGroupedListV2Props> = props => {
   const renderHeader = (item: IHeaderGroupedItem, flattenedIndex: number): React.ReactNode => {
     const group = item.group;
 
+    let ariaProps;
+    if (role === 'treegrid') {
+      // GroupedList default role
+      ariaProps = {
+        ariaLevel: group.level ? group.level + 1 : 1,
+        ariaSetSize: groups ? groups.length : undefined,
+        ariaPosInSet: item.groupIndex,
```

Comment:
This looks good for the header, though the non-expanding rows within a treegrid also need level & posinset info, rather than `rowindex`.

Reply:
It turns out `GroupedListV1` is accomplishing this by casting `onRenderCell` to `any` and then passing a final argument, `group` to this render callback. This argument is then used by `DetailsRow` to properly set the aria values. I've updated the types for `GroupedListV2` to include the `group` argument in `onRenderCell` and `GroupedListV2` passes this through so aria values are set properly.